app/controllers/admin/invoices_controller.rb

```diff
@@ -7,5 +7,16 @@ def index
   def show
     @invoice = Invoice.find(params[:id])
   end
+
+  def update
+    @invoice = Invoice.find(params[:id])
+    @invoice.update(invoice_params)
+    @invoice.save
```

Comment:
This method works great, but I don't believe the `save` method is necessary here as `update` should save for us.